internal/proxytag/sigils.go

```diff
@@ -129,3 +134,11 @@ func Sigils(message string) (Match, error) {
 		Body:         body,
 	}, nil
 }
+func Reverse(s string) string {
```

Comment:
This will produce incorrect output for multi-byte UTF-8 runes, try:

```go
func Reverse(s string) string {
    runes := []rune(s)
    for i, j := 0, len(runes)-1; i < j; i, j = i+1, j-1 {
        runes[i], runes[j] = runes[j], runes[i]
    }
    return string(runes)
}
```

Reply:
👍